packages/mui/src/date-time-pickers/wrapper.tsx

```diff
@@ -29,19 +34,38 @@ type PickerProps<
 type Props<P extends PickerTypes, In, Out> = SubsetPartial<
   PickerProps<P, In, Out>,
   'renderInput'
->;
+> &
+  (P extends typeof TimePicker ? unknown : { multiLocale?: boolean });
```

Comment:
change name to multiCalendar

Reply:
No, multiLocale is good, because it also changes the language for month and weekday names!